src/main/app/case/answers/possibleAnswers.ts

```diff
@@ -37,7 +39,14 @@ export const getAllPossibleAnswers = (caseState: Partial<Case>, steps: Step[]):
     return fields;
   };
 
-  return getPossibleFields(sequenceWithForms[0]);
+  const applicant1Fields = getPossibleFields(sequenceWithForms[0]);
```

Comment:
i feel like you could reduce this and basically pass the individual sequences to the function.

Reply:
```suggestion
  const possibleAnswers = [...getAllPossibleAnswers(userCase, stepsWithContentApplicant1), ...getAllPossibleAnswers(userCase, stepsWithContentApplicant2)];
```
couldnt edit where it should go but i thought where the function is called.

Reply:
I see what you mean, will just need to control when app2 one is included so that it is only called when the application is joint.